Homework3/DataAccess.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace start
+{
+    public static class DataAccess
+    {
+        private static readonly string fileName = $"Users.txt";
+
+        private static string GetStringsFromFile()
```

Comment:
It's more than just a string, name method after logical data they return, not how it looks like (semantics).

Reply:
Move this method under the ones that consume it.

Reply:
Renamed to GetUserCredentialsFromFile and moved down.